packages/core/src/app/checkout/Checkout.scss

```diff
@@ -0,0 +1,102 @@
+@import '../ui/Base';
+
+$color-secondary: #757575;
+$color-link: #476BEF;
+$fontFamily-montserrat: 'Montserrat';
```

Comment:
These values appear to have a global effect.
Perhaps we should manage them in `packages/core/src/scss/settings/global/color/_color.scss` and `packages/core/src/scss/settings/global/typography/_typography.scss`.